### PR TITLE
Remove improper handling for the refresh command

### PR DESCRIPTION
**Pull request by s-k-z**
Currently, typing `help refresh` does not work as expected. These lines of code improperly intercept the keyword refresh and alter the inputted command in undesirable ways for all other commands as well such that any command of the form `command refresh` will attempt to run the refresh command instead.

Currently:
```
> help refresh

help cannot be refreshed.

> genie refresh

genie cannot be refreshed.
```

With this change:
```
> help refresh

refresh all | [status | effects] | [gear | equip | outfit] | inv | camp | storage | [familiar | terarrium] | stickers | quests | shop - resynchronize with KoL.

> refresh genie

genie cannot be refreshed.

> genie refresh

(no output)
```

### CODE REVIEW

**Comment by gausie** [comment]
Presumably this meant you could do `status refresh` instead of `refresh status`.

I agree that is silly and should go away. Let's merge this. Can you add a regression test please?

File: test/net/sourceforge/kolmafia/textui/command/HelpCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  @Test
+  void Refresh() {
+    String output = execute("refresh");
+
+    assertContinueState();
+    assertThat(
+        output,
+        containsString(
```
Comment (gausie) [inline test/net/sourceforge/kolmafia/textui/command/HelpCommandTest.java:28]: Tiny thing but could you change this to just be startsWith `refresh all |`. That would reasonably cover the regression while also not make us have to change this test every time we add a new option to refresh

Comment (gausie) [reply test/net/sourceforge/kolmafia/textui/command/HelpCommandTest.java]: Can you make this `startsWith` though, I'm so sorry!

Comment (s-k-z) [reply test/net/sourceforge/kolmafia/textui/command/HelpCommandTest.java]: Sure, I overlooked that at first, haha.

**Review by gausie — APPROVED** [review]

**Pull request merged**